src/wallet/crypter.cpp

```diff
@@ -173,7 +173,7 @@ bool CCryptoKeyStore::SetCrypted()
     LOCK2(cs_KeyStore, cs_SpendingKeyStore);
     if (fUseCrypto)
         return true;
-    if (!(mapKeys.empty() && mapSpendingKeys.empty()))
+    if (!(mapKeys.empty() && mapSpendingKeys.empty() && mapSaplingSpendingKeys.empty()))
```

Comment:
`mapSpendingKeys` should be renamed to `mapSproutSpendingKeys`.

Reply:
I was hoping to avoid that large rename for now, but I'll go rip off the plaster 😂

Reply:
Oh, turns out it's not a large rename after all!